CalculatorExercises/1-CalculatorBasic/StringHelper.java

```diff
@@ -0,0 +1,7 @@
+public class StringHelper {
+
+    public static String removeWhitespace (String string) {
```

Comment:
you can
return string.replaceAll("\\s","");

Reply:
or if it is just one line maybe you can remove that file and use that method directly in main